pkg/segment/utils/segconsts.go

```diff
@@ -1032,10 +1051,10 @@ func (e *CValueEnclosure) AsBytes() []byte {
 			return []byte("true")
 		}
 		return []byte("false")
-	case SS_DT_SIGNED_NUM, SS_DT_SIGNED_32_NUM, SS_DT_SIGNED_16_NUM, SS_DT_SIGNED_8_NUM:
+	case SS_DT_SIGNED_NUM:
 		buf := make([]byte, 0, 20)
 		return strconv.AppendInt(buf, e.CVal.(int64), 10)
-	case SS_DT_UNSIGNED_NUM, SS_DT_USIGNED_32_NUM, SS_DT_USIGNED_16_NUM, SS_DT_USIGNED_8_NUM:
+	case SS_DT_UNSIGNED_NUM:
```

Comment:
Why do we want this change?

Reply:
Ok that makes sense why you'd want that, but the function is just `AsBytes()`, so it should work for any valid `CValueEnclosure` type. I think with your latest commits you're not using this anymore, so you can revert your changes. Actually I think no code is using this function, so you could even delete the function if you don't think we'll need it in the future

Reply:
Yes, I have removed the function.